src/pages/RepoPage/BundlesTab/BundleContent/AssetsTable/AssetsTable.tsx

```diff
@@ -161,7 +161,7 @@ const createColumns = (totalBundleSize: number | null) => [
             />{' '}
             <ChangeOverTime
               change={value?.change?.size.uncompress}
-              hasMeasurements={value.measurements.length > 0 ?? false}
+              hasMeasurements={value.measurements.length > 0}
```

Comment:
This was something that Vite kept bringing up as an optimization so finally threw it in.